### ISSUE TITLE
Update wording in discard dialog to make it more clear all files will be discarded

### ISSUE THREAD

**Issue by jonest** [microsoft/vscode#178166]
### TL;DR Consider rewording the Discard All dialog to clarify that it's ALL files, not ALL changes in X files

I just had this happen for the first time where I had selected several files after switching branches and the .gitignore wasn't ignoring the new files that were generated. I went to select all the files I didn't want (50+), leaving the changes I had made on the branch I had just switched to. I clicked the wrong discard button and accidentally discarded all my fresh changes I was working on.

Here's an example of the current wording:
![image](https://user-images.githubusercontent.com/3075414/227299600-3e554c73-68cc-4992-a76a-72b9b84c8798.png)

When I saw this, I thought "Yes, I want to discard all the CHANGES to the X files I selected", but what I think would be better wording is, when the "discard all" button is clicked, to say "Discard changes in ALL files" to make it clear that it's not the X you selected, but ALL files. I think that if I had seen it say "ALL files" there would have been a much higher probability that I would have realized my mistake.

For reference, this is what it says when the correct button is clicked:
![image](https://user-images.githubusercontent.com/3075414/227311311-274cf37d-9ed4-4345-b158-29efe6f033bf.png)

The wording is quite similar, so when I saw a file number, I just quickly skipped past the rest of the wording to discard the changes, since, as I mentioned, yes, I want to discard all the changes in all the files I selected and I know that it's irreversible. (Maybe the wording should be updated here as well to add that disclaimer?)

Sure, I could have looked to see that the change count in the dialog wasn't the same number as the change count in the git panel, but changing this wording seems like a more clear way to help keep users working quickly from shooting themselves in the foot, like I just did 😬
